### PR TITLE
Update the doc for pandas udf on databricks

### LINKED ISSUE #2024: [DOC] Document "spark.python.daemon.module rapids.daemon_databricks" when running on databricks
## Report incorrect documentation

**Location of incorrect documentation**
Provide links and line numbers if applicable.
https://github.com/NVIDIA/spark-rapids/blob/branch-0.5/docs/get-started/getting-started-databricks.md

**Describe the problems or issues found in the documentation**
A clear and concise description of what you found to be incorrect.

When using pandas cuDF UDF on databricks, it fails with below error:
```
File "/databricks/jars/rapids-4-spark_2.12-0.4.1.jar/rapids/daemon.py", line 140, in manager
TypeError: worker() missing 1 required positional argument: 'executor_username'
```

This is because we have 2 python daemons -- `daemon.py` and `daemon_databricks.py` under python/rapids directory.
In `worker.py`: `worker_main(sock_file, outfile)` 
In `daemon_databricks.py`: `code = worker(sock, authenticated, executor_username)`
We should use `daemon_databricks` as the python daemon.

**Steps taken to verify documentation is incorrect**
List any steps you have taken:

Enable pandas cuDF UDF on databricks and run a sample udf such as
```@pandas_udf('double')
def cudf_pandas_plus_one(v: pd.Series) -> pd.Series:  
  gpu_series = cudf.Series(v)
  gpu_series = gpu_series + 1
  return gpu_series.to_pandas()

df.withColumn('v2', cudf_pandas_plus_one(df.v)).show()
```

**Suggested fix for documentation**
Detail proposed changes to fix the documentation if you have any.

In `getting-started-databricks.md`, specify if users needs to use pandas cuDF UDFs,
we should specify below in `Advanced Options`-> `Spark` tab by adding below:
`spark.python.daemon.module rapids.daemon_databricks` 

---

## Report needed documentation

**Report needed documentation**
A clear and concise description of what documentation you believe it is needed and why.

**Describe the documentation you'd like**

**Steps taken to search for needed documentation**
List any steps you have taken:

### PR DESCRIPTION
**Pull request by firestarman**
Update the doc for pandas udf on databricks, since an additional config is required to run Pandas UDF with our plugin on DB.

closes #2024

Signed-off-by: Firestarman <firestarmanllc@gmail.com>

### CODE REVIEW

**Comment by firestarman** [comment]
ci build is not required i think.

**Comment by firestarman** [comment]
build

**Comment by tgravescs** [comment]
@viadea does this match what you had to do?

**Comment by viadea** [comment]
> @viadea does this match what you had to do?

@tgravescs @firestarman  Do you think if we need to specify below parameter if they want to make pandas UDF work? or you prefer we specify this in the UDF doc? 

`spark.executorEnv.PYTHONPATH /databricks/jars/rapids-4-spark_2.12-0.4.1.jar:/databricks/spark/python`

Other than that, it looks good to me. thanks @firestarman for helping explain this.

**Comment by firestarman** [comment]
@viadea I added the cofig in the Databricks doc, since it has specific value for Databricks.

**Comment by firestarman** [comment]
build

**Comment by firestarman** [comment]
@viadea  is it ok for you ? if yes, could you approve it ?

File: docs/get-started/getting-started-databricks.md [hunk 1]
```diff
@@ -85,6 +85,21 @@ cluster.
...
     ![Spark Config](../img/Databricks/sparkconfig.png)
 
+   When you are going to run Pandas UDFs with GPU support from the plugin, at least three additional options
+    as below are required. The `spark.python.daemon.module` option is to choose the right daemon module
+    of python for Databricks. On Databricks, the python runtime requires different parameters than the
+    Spark one, so this dedicated python deamon module `rapids.daemon_databricks` is created and should
+    be specified here. Set the config [`spark.rapids.sql.python.gpu.enabled`](../configs.md) to true to
+    enable GPU support for python. And add the path of the plugin jar (supposing it is placed under
+    `/databricks/jars/`) to the `spark.executorEnv.PYTHONPATH` option. More details please go to
+    [**GPU Scheduling For Pandas UDF**](../additional-functionality/rapids-udfs.md#gpu-scheduling-for-pandas-udf)
```
Comment (sameerz) [inline docs/get-started/getting-started-databricks.md:95]:
```suggestion
    If running Pandas UDFs with GPU support from the plugin, at least three additional options
    are required. The `spark.python.daemon.module` option is to choose the right daemon module
    of python for Databricks. On Databricks, the python runtime requires different parameters than the
    Spark one, so a dedicated python deamon module `rapids.daemon_databricks` is created and should
    be specified here. Set the config [`spark.rapids.sql.python.gpu.enabled`](../configs.md#sql.python.gpu.enabled) 
    to `true` to enable GPU support for python.  Add the path of the plugin jar (supposing it is placed under
    `/databricks/jars/`) to the `spark.executorEnv.PYTHONPATH` option.  For more details please go to
    [**GPU Scheduling For Pandas UDF**](../additional-functionality/rapids-udfs.md#gpu-scheduling-for-pandas-udf)
```

Comment (firestarman) [reply docs/get-started/getting-started-databricks.md]: Thanks for review. Updated

**Review by viadea — APPROVED** [review]

**Comment by firestarman** [comment]
build

**Pull request merged**